mantle/cmd/ore/gcloud/upload.go

```diff
@@ -153,12 +157,7 @@ func runUpload(cmd *cobra.Command, args []string) {
 			switch ans {
 			case "y", "Y", "yes":
 				fmt.Println("Overriding existing image...")
-				_, pending, err = api.CreateImage(&gcloud.ImageSpec{
-					Name:        imageNameGCE,
-					Family:      uploadImageFamily,
-					SourceImage: imageStorageURL,
-					Description: uploadImageDescription,
-				}, true)
+				_, pending, err := api.CreateImage(spec, true)
```

Comment:
Note that the change from `=` to `:=` here conflicts with the error detection fix in #1454.

Reply:
Nope, I'm wrong, sorry.